.github/workflows/actionlint.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
```

Comment:
there is a workflow named CI already: https://github.com/sveltejs/kit/blob/master/.github/workflows/ci.yml#L1

Reply:
Yes, but then this job would fire on each PR; not only those that change relevant files which is why I took this approach instead.

Reply:
This job would need a different name to tell them apart

Reply:
Ah, my bad. Apologies